### PR TITLE
[RHACS] ROX-19179: Updating roxctl netpol commands

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.2

Issue:
https://issues.redhat.com/browse/ROX-19179

Link to docs preview:
https://64300--docspreview.netlify.app/openshift-acs/latest/operating/manage-network-policies#generate-build-time-network-policies_manage-network-policies

SME review:
- [x] SME has approved this change.

Additional information:
Merge into `rhacs-docs` and cherry-pick into:
- `rhacs-docs-4.2`

### CODE REVIEW

File: modules/generate-build-time-network-policies.adoc [hunk 1]
```diff
@@ -41,17 +41,17 @@ Following are some examples that match the pattern:
 +
 [source,terminal]
 ----
-$ roxctl generate netpol -h
+$ roxctl netpol generate -h
 ----
 . Generate the policies by using the `generate netpol` command:
```
Comment (zivnevo) [inline modules/generate-build-time-network-policies.adoc:46]:
```suggestion
. Generate the policies by using the `netpol generate` command:
```

**Review by zivnevo — APPROVED** [review]
LGTM

**Comment by xenolinux** [comment]
LGTM

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead. Thanks!

**Comment by abhatt-rh** [comment]
> LGTM
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead. Thanks!

@agantony @xenolinux: Peer review LGTM!

**Pull request merged**